src/objectExplorer/objectExplorerService.ts

```diff
@@ -244,6 +244,37 @@ export class ObjectExplorerService {
         return [new AddConnectionTreeNode()];
     }
 
+    /**
+     * Removes password from a saved profile
+     */
+    private async removeProfilePassword(connection: IConnectionCredentials): Promise<void> {
```

Comment:
This seems like it should be something done in the connectionStore class since that should really be handling all the connection-specific logic.